docs/release-notes.md

```diff
@@ -4,7 +4,9 @@ nav_order: 8
 
 # Release notes
 
-## Upcoming Afterburn 5.7.0 (unreleased)
+## Upcoming Afterburn 5.8.0 (unreleased)
```

Comment:
We usually include the sub boxes  aka

- Major changes:
- Bug fixes: 
- etc.